Retouching on the converter documentations. (#239)
PowerGridModel/power-grid-model-io: docs/converters/vision_converter.md

Custom mapping files are supported via passing the file directory to the constructor of the converter.

## Load rate of elements 

Certain `elements` in Vision, i.e., appliances like transformer loads and induction motor have a result parameter of load rate.
In Vision, load rates are calculated without considering the simultaneity factors of connected nodes.
So we may observe a variation in power inflow/outflow result (i.e., P, Q and S) due to different simultaneity factors. But the load rate always corresponds to `simultaneity of loads=1`.

When we make conversion to PGM, the input data attributes of PGM for loads like `p_specified` and `q_specified` are modified as per simultaneity. The resulting loading then takes simultaneity into account. 
**Hence, the loading of such elements may not correspond to the load rate obtained in Vision**

## Transformer load modelling

power-grid-model-io converts the transformer load into a individual transformer and a load for usage in power-grid-model. 

> In Vision, the modelling of an transformer load may seem to be different from an individual transformer and load.

Reviewer: ```suggestion
As far as we can tell, in Vision, the modeling of a transformer load is different from an individual transformer and load.
```

There is a minor difference in both the reactive power consumed and generated. 

> This could also correspond to a minor voltage deviation in the results.

Author: This whole section is quite vague. Was the original intention to inform users of potential cases with minor error?

Reviewer: it's mainly to explain that there's deviations from the literature in Vision. we follow the literature conventions

Author: @petersalemink95 Perhaps you could give us some insight here?

> This could also correspond to a minor voltage deviation in the results.

Reviewer: ```suggestion
This may correspond to a minor voltage deviation in the results.
```
